src/main/resources/db/migration/V1_4__add_prison_code_permitted_prisoner.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE permitted_prisoner ADD prison_code VARCHAR(3);
+UPDATE permitted_prisoner set prison_code = 'NA' where prison_code IS NULL;
```

Comment:
What prison is NA ?

Reply:
setting it to NA for the timebeing before setting it to not nullable. Maybe updating to Foston Hall code makes more sense.